docs/api.md

```diff
@@ -671,12 +671,51 @@ const result = await page.evaluate(() => {
 console.log(result); // prints "56"
 ```
 
+Passing arguments to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate((name, age, location) => {
+  return {
+    name,
+    age,
+    location
+  };
+}, name, age, location);
+
+console.log(information); // prints "{ name: 'Thomas', age: 29, location: 'Paris' }"
+```
+
+Passing both ```pageFunction``` and ```...args``` to ```page.evaluate``` as parameters.
+
+```js
+function manipulateDOM(arg){
+  // manipulate DOM logic
+  return `puppeteer was here and received ${arg}`;
+}
+
+const information = await page.evaluate(manipulateDOM, 42);
+
+console.log(information); // prints: "puppeteer was here and received 42";
+```
+
 A string can also be passed in instead of a function.
 
 ```js
 console.log(await page.evaluate('1 + 2')); // prints "3"
```

Comment:
let's incorporate template string example here and demonstrate string interpolation:
```js
const x = 10;
console.log(await page.evaluate(`1 + ${x}`)); // prints "11"
```